modules/malltvAnalyticsAdapter.js

```diff
@@ -0,0 +1,188 @@
+import {ajax} from '../src/ajax.js'
+import adapter from '../src/AnalyticsAdapter.js'
+import CONSTANTS from '../src/constants.json'
+import adapterManager from '../src/adapterManager.js'
+import {getGlobal} from '../src/prebidGlobal.js'
+import {logInfo, logError, deepClone} from '../src/utils.js'
+
+const analyticsType = 'endpoint'
+export const ANALYTICS_VERSION = '1.0.0'
+export const DEFAULT_SERVER = 'https://central.mall.tv/analytics'
+
+const {
+  EVENTS: {
+    AUCTION_END,
+    BID_TIMEOUT
+  }
+} = CONSTANTS
+
+export const BIDDER_STATUS = {
+  BID: 1,
+  NO_BID: 2,
+  BID_WON: 3,
+  TIMEOUT: 4
+}
+
+export const getCpmInEur = function (bid) {
+  if (bid.currency === 'EUR') {
+    return bid.cpm
+  } else {
+    return bid.getCpmInNewCurrency('EUR')
```

Comment:
Actually, you need to add a condition here in the instance that getCpmInNewCurrency does not exist. Please check that it exists as a function before running

Reply:
Also updating here should correct your unit tests and allow them to pass